fix: checkout -> check out (#2116)
getsentry/sentry-docs: src/docs/clients/javascript/integrations.mdx

## What are plugins? {#what-are-plugins}

In Raven.js, plugins are little snippets of code to augment functionality for a

> specific application/framework. It is highly recommended to check out the list

Reviewer: ```suggestion
specific application/framework. We highly recommended you check out the list
```
